bin/console

```diff
@@ -1,27 +1,16 @@
 #!/usr/bin/env php
 <?php
 
-use Symfony\Bundle\FrameworkBundle\Console\Application;
+use eLife\Annotations\AppKernel;
 use Symfony\Component\Console\Input\ArgvInput;
-use Symfony\Component\Debug\Debug;
 
-// if you don't want to setup permissions the proper way, just uncomment the following PHP line
-// read https://symfony.com/doc/current/setup.html#checking-symfony-application-configuration-and-setup
-// for more information
-//umask(0000);
+require_once __DIR__.'/../vendor/autoload.php';
 
+umask(002);
 set_time_limit(0);
 
-require __DIR__.'/../vendor/autoload.php';
-
 $input = new ArgvInput();
 $env = $input->getParameterOption(['--env', '-e'], getenv('SYMFONY_ENV') ?: 'dev');
```

Comment:
we haven't found a way to provide an env variable to long-running processes, so for now `--env` is the best way